src/test/java/seedu/saveit/logic/commands/AddCommandTest.java

```diff
@@ -71,6 +82,128 @@ public void execute_duplicateIssue_throwsCommandException() throws Exception {
         addCommand.execute(modelStub, commandHistory);
     }
 
+    @Test
+    public void execute_issueLevelAddIssue_throwsCommandException() throws Exception {
+        Issue validIssue = new IssueBuilder().build();
+        AddCommand addCommand = new AddCommand(validIssue);
+        ModelStub modelStub = new ModelStubAcceptingIssueAdded();
+        Directory issueLevel = getCustomizedIssueLevel(INDEX_FIRST_ISSUE);
+        modelStub.resetDirectory(issueLevel);
+
```

Comment:
Since you use a lot of `modelStub` and `resetDirectory`, you may want to extract them into a separate method `setUp()` or so, to make your code neater.